core/libraries/rest_api/calculations/Attendee.php

```diff
@@ -2,14 +2,8 @@
 
 namespace EventEspresso\core\libraries\rest_api\calculations;
 
-use EE_Attendee;
-use EE_Error;
-use EEM_Attendee;
-use EventEspresso\core\exceptions\InvalidDataTypeException;
-use EventEspresso\core\exceptions\InvalidInterfaceException;
-use EventEspresso\core\libraries\rest_api\calculations\Base as Calculations_Base;
-use EventEspresso\core\libraries\rest_api\controllers\model\Base;
-use InvalidArgumentException;
+use EventEspresso\core\libraries\rest_api\calculations\Base as AttendeeCalculationsBase;
+use EventEspresso\core\libraries\rest_api\controllers\model\Base as AttendeeControllerBase;
```

Comment:
I had to make these modifications to every Calculations class because of a related fatals occurring in unit tests run in PHP5.6 and PHP5.5 (exposed on travis).  I suspect this is because of the new schema generation being tested for `_calculated_fields` and thus something like this wasn't exposed prior to this branch. 

I suspect this is related to [this php bug](https://bugs.php.net/bug.php?id=66773) and _may_ also be related to the `class_implements` check I'm doing which surfaces that bug.  At any rate, the solution is to have unique aliases across the static classes as I implemented.  So you'll see this in all the Calculations classes.